app/models/fee/base_fee_type.rb

```diff
@@ -25,7 +25,7 @@ def initialize(message = DEFAULT_MSG)
   end
 
   class BaseFeeType < ActiveRecord::Base
-    ROLES = %w[lgfs agfs].freeze
+    ROLES = %w[lgfs agfs agfs_scheme_9 agfs_scheme_10].freeze
```

Comment:
💭  Potentially remove this in the future ⏩  in favour of association with a given fee scheme.